src/main/java/com/quizter/repository/PasswordRepository.java

```diff
@@ -0,0 +1,14 @@
+package com.quizter.repository;
+
+import com.quizter.entity.PasswordResetToken;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+@Repository
+public interface PasswordRepository extends CrudRepository<PasswordResetToken, Long> {
+
+    PasswordResetToken save(PasswordResetToken passwordResetToken);
```

Comment:
Looks like you override standard save method from CrudRepository

Reply:
Ok